### ISSUE TITLE
Can’t restart on Windows

### ISSUE THREAD

**Issue by Senyoret1** [skycoin/skywire#1641]
**Describe the bug**
The API endpoint for restarting a visor does not work on Windows.

**Environment information:**

- OS: Windows
- Platform: Windows 10

**Steps to Reproduce**

1. Run a visor.
2. Use the `Reboot` option on the UI (you can also use the API if you want).
3. The operation will fail with this error: “Exec: "/bin/sh": file does not exist.”

**Actual behavior**
The operation does not work.

**Expected behavior**
The operation should work.

**Comment by 0pcom** [comment]
I'm not sure is `visor restart` or `visor reload` were ever implemented correctly.

We used to have a way that the visor would restart for updates, I think.

But all the updater code was removed, and what was left were some api endpoints that were added as cli commands just because they existed.

In the packages, currently, the process running via a systemd service, it's automatically restarted on shutdown (currently) regardless of exit status because there were some instances of exit without errors.
